lib/filter.g

```diff
@@ -171,6 +171,60 @@ BIND_GLOBAL( "InstallTrueMethodNewFilter", function ( tofilt, from )
 end );
 
 
+#############################################################################
+##
+#F SuspendMethodReordering()
+#F ResumeMethodReordering()
+#F ResetMethodReordering()
```

Comment:
For consistency with other files, there should be two spaces after each `#F`, and also `()` should be `( )` (but we'll survive w/o this changed, too ;-)